test/evaluate/folding.cc

```diff
@@ -39,7 +47,6 @@ struct TestGetScalarConstantValue {
 };
 
 int main() {
-  using TestTypes = AllIntrinsicTypes;
-  RunOnTypes<TestGetScalarConstantValue, TestTypes>::Run();
+  RunOnTypes<TestGetScalarConstantValue, AllIntrinsicTypes>::Run();
```

Comment:
Are there tests for the new functionality?

Reply:
No, they were actually added in a PR fixing GetScalarConstantValue. The changes appearing here are unfortunate left over modifications from PR #275. I should have copied the original file after the cherry-pick from PR #275.
In PR #275, I had added tests for some intrinsic functions folding. For now, I have removed them until the interface is stable.